### PR TITLE
python_base: add firefox for acceptance tests

### PR DESCRIPTION
**Pull request by rikirenz**
Adding firefox, in order to have a visible browser when we run acceptance tests

### CODE REVIEW

File: python_base/Dockerfile [hunk 1]
```diff
@@ -40,7 +40,8 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
         postgresql-devel \
         python-pip \
         python-virtualenv \
-        wget && \
+        wget \
+        firefox && \
```
Comment (jacquerie) [inline python_base/Dockerfile]: These dependencies are in alphabetical (lexicographic) order.

File: python_base/Dockerfile [hunk 2]
```diff
@@ -40,7 +40,8 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
         postgresql-devel \
         python-pip \
         python-virtualenv \
-        wget && \
+        wget \
```
Comment (jacquerie) [inline python_base/Dockerfile]: Broken `&&` chain.

File: python_base/Dockerfile [hunk 3]
```diff
@@ -69,7 +70,8 @@ RUN virtualenv /tmpvenv -p python${INSPIRE_PYTHON_VERSION} && \
     deactivate && \
     mv /tmpvenv/src /src-cache && \
     rm -rf /tmpvenv && \
-    mkdir /code
+    mkdir /code && \
+    dbus-uuidgen > /etc/machine-id
```
Comment (jacquerie) [inline python_base/Dockerfile]: If anyone else is wondering, apparently `firefox` needs a well-formed `machine-id` to start.

**Comment by rikirenz** [comment]
Ready for review

File: python_base/Dockerfile (hunk 2 above)
Comment (david-caro) [inline python_base/Dockerfile]: That's why usually I put the && in the beginning... way more readable imo

File: python_base/Dockerfile (hunk 3 above)
Comment (david-caro) [inline python_base/Dockerfile]: I think that the issue is more related to using DBus or similar when connecting to the X server. But yep, what you get is that firefox is not running.

**Comment by david-caro** [comment]
Haven't actually checked if the machine-id file is correctly created and I don't find the log trace on the travis build page, looks ok, if nobody tests it and merges it before, I'll test later.

File: python_base/Dockerfile (hunk 3 above)
Comment (kaplun) [inline python_base/Dockerfile]: Interesting!

**Comment by kaplun** [comment]
Just looking at the code, LGTM

**Comment by david-caro** [comment]
It works ok for me locally

**Pull request merged**